R/jjcsformats.R

```diff
@@ -0,0 +1,230 @@
+#' Title: Function factory for xx style formatting
+#'
+#' @description A function factory to generate formatting functions for value
+#' formatting that support the xx style format and control the rounding method
+#' @importFrom tidytlg roundSAS
+#' @param roundmethod (`string`)\cr choice of rounding methods. Options are:
+#'   * `SAS`: the underlying rounding method is `tidytlg::roundSAS`, where \cr
+#'   roundSAS comes from this Stack Overflow post https://stackoverflow.com/questions/12688717/round-up-from-5
+#'   * `R`: the underlying rounding method is `round`
+#'
+#' @return `jjcs_format_xx_fct()` format function that can be used in rtables formatting calls
+#' @export
+#'
+#' @family JJCS formats
+#' @examples
+#' jjcsformat_xx_SAS <- jjcs_format_xx_fct(roundmethod="SAS")
+#' jjcsformat_xx <- jjcsformat_xx_SAS
+#' format_value(c(1.453),jjcsformat_xx("xx.xx"))
+#' format_value(c(1.453,2.45638),jjcsformat_xx("xx.xx (xx.xxx)"))
+#'
+jjcs_format_xx_fct <- function (roundmethod=c("SAS","R"))
```

Comment:
```suggestion
jjcs_format_xx_fct <- function(round_method = c("SAS", "R"))
```
We use snake case fore all our variables and fnc. Camel case ONLY for Classes